src/test/java/com/jcabi/github/PullCommentTest.java

```diff
@@ -188,4 +189,24 @@ public void updatesBody() throws Exception {
             Json.createObjectBuilder().add(BODY, value).build()
         );
     }
+
+    /**
+     * PullComment.Smart can retrieve who is the comment author.
+     * @throws Exception If a problem occurs.
+     */
+    @Test
+    public void getAuthor() throws Exception {
```

Comment:
`retrievesAuthor` would be a better name (`getAuthor` sounds too much like a getter :)